addons/radar/functions/fnc_getGPS.sqf

```diff
@@ -0,0 +1,8 @@
+#include "script_component.hpp"
+params ["_unit"];
+
+private _array = (assignedItems _unit) arrayIntersect GVAR(gpsWhitelist);
+if (_array isNotEqualTo []) exitWith {
+    _array select 0
+};
+""
```

Comment:
I would say use a param here. would be more simple.
```suggestion
_array param [0, ""];
```

Reply:
that function is a copy of the compass one, so might need to be changed there as well.

Reply:
Thought about it a little longer. Yeah that makes sense. @Malbryn could you apply the change and also change the getCompass function?